app/assets/javascripts/application.js

```diff
@@ -14,5 +14,10 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
-//= require jquery
-//= require bootstrap-sprockets
\ No newline at end of file
+
+//= require landing_page/jquery.js
+//= require landing_page/bootstrap.min.js
+//= require landing_page/custom.js
+//= require landing_page/jquery.sticky.js
+//= require landing_page/wow.min.js
+//= require landing_page/owl.carousel.min.js
```

Comment:
It's not a good place for requires like this. See `require_tree .`. This thing means "require all files under current folder". You should place all landing page specific requires in the separate file. It will allow us to create custom bundle specifically for landing page in the future.